onnxruntime/core/session/provider_registration.cc

```diff
@@ -86,8 +86,8 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
 #endif
   } else if (strcmp(provider_name, "WEBNN") == 0) {
 #if defined(USE_WEBNN)
-    std::string deviceType = options->value.config_options.GetConfigOrDefault("deviceType", "2");
-    std::string powerPreference = options->value.config_options.GetConfigOrDefault("powerPreference", "0");
+    std::string deviceType = options->value.config_options.GetConfigOrDefault("deviceType", "cpu");
```

Comment:
we have 2 different places in the code to set default value of the configs. this may cause potential inconsistency.

is there a way to put default value to only one place?

Reply:
The other place is Line 29 in webnn_provider_factory.cc , in case you don't find it.

Reply:
Good point! These should be duplicated. I will remove one of them.